README.md

```diff
@@ -115,7 +114,6 @@ unnecessary and will be ignored if you are using a different backend).
  Mobility.configure do |config|
    config.default_backend = :key_value
    config.accessor_method = :translates
-   config.query_method    = :i18n
 +  config.default_options[:dirty] = true
 +  config.default_options[:type]  = :string
```

Comment:
Haha this is a mistake by github. These lines are not added in this PR, they just have a `+` at the head of the line and happen to follow a (correctly) deleted line.